### PR TITLE
chore(TNLT-000): remove android and IOS app

### PR DESCRIPTION
**Pull request by mikefieldmay**
We no longer need to publish for IOS and Android.
This PR begins to remove those build steps and files from Times Components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,21 +30,6 @@
     "storybook:test-build": "build-storybook -c .storybook -s ./dist/public",
     "storybook:publish": "./lib/publish_storybook.sh",
     "storybook-native": "react-native start --config ./storybook-native/metro.config.js",
```
Comment (flashcheeks) [inline package.json:32]: Do we need to remove `storybook-native` too

File: README.md [hunk 2]
```diff
@@ -15,7 +15,6 @@ We require MacOS with [Node.js](https://nodejs.org) (for specific version please
 
 Native development
```
Comment (flashcheeks) [inline README.md:16]: Are we removing native completely?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -5,14 +5,14 @@ Following these guidelines helps to get issues organised and PRs merged faster!
 
 ## Core Ideas
 
-* Components should work across all platforms (web, iOS, Android) to the same
+* Components should work across all platforms (web, iOS) to the same
```
Comment (flashcheeks) [inline .github/CONTRIBUTING.md:8]: Are we keeping iOS?

File: .eslintignore [hunk 4]
```diff
@@ -5,5 +5,3 @@ dist
 storybook-static
 storybook-native
```
Comment (flashcheeks) [inline .eslintignore:6]: Do we need to remove `storybook-native`

**Review by flashcheeks — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mikefieldmay) [reply package.json:32]: Good spot

**Pull request merged**